### DISCUSSION TITLE
Permissions header bar is sticky

### DISCUSSION THREAD

**Discussion by TomSmedley** [directus/directus#12702] [Feature Requests]
When you have a lot of collections, permissions become a little confusing as you scroll down as you lose track of which box does what.
![image](https://user-images.githubusercontent.com/95056193/162930019-77cf2c27-6846-47b4-8beb-32aba0ffb8a1.png)

It would be better if this was a sticky element so you know what you are editing at all times.

**Comment by rijkvanzanten** [comment]
This was added in https://github.com/directus/directus/pull/12704
